src/kit/Surface.module.scss

```diff
@@ -0,0 +1,140 @@
+@use 'sass:map';
+@use 'Elevation/index.module.scss';
+
+$levels-light: (
+  0: var(--elevation-0-bg-light),
```

Comment:
I must have missed this in the elevation PR but there shouldn't be different variables for light and dark.

Reply:
Ah you're right. I was getting the variables from `Elevation/index.module.scss` instead of `Theme/themes.ts`. I think we can delete that Elevation scss file then.